tests/data/apply_filters.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SzepeViktor\PHPStan\WordPress\Tests;
+
+use function apply_filters;
+use function PHPStan\Testing\assertType;
+use stdClass;
+
+$value = apply_filters('filter','Hello, World');
+assertType('mixed', $value);
+
+/**
+ * Single type.
+ *
+ * @param string $foo Hello, World.
+ */
+$value = apply_filters('filter',$foo);
```

Comment:
The thing is value is usually a constant: a number or boolean. How to merry `@param` and a constant?

Reply:
Good point, I added tests for this.